src/h2/create-table.sql

```diff
@@ -2,23 +2,21 @@
 -- You may use, distribute and modify this code under the
 -- terms of the Apache License v2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt).
 
-create table BOUNDING_BOX(
+create table if not exists BOUNDING_BOX(
   file_name varchar(255) primary key,
   top int not null,
   left int not null,
   bottom int not null,
   right int not null,
   width int not null,
-  height int not null
+  height int not null,
+  dataset varchar(255) default 'UNKNOWN'
```

Comment:
Are the values of possible datasets known a priori? Reason I ask is, if it is, it's worth considering Postgres enums for that extra safety net at the DB level.